### PR TITLE
Added functinality to edit autonag and test case to check autoneg edit

### PR DESCRIPTION
**Pull request by vivek-metricdust**
fixes [#56](https://github.com/STORDIS/orca_nw_lib/issues/56) orca_backend

added functonality where autoneg is configurable in interfaces.
added testcases to check the autoneg and change the autoneg to off and on

### CODE REVIEW

File: network/test/test_interface.py [hunk 1]
```diff
@@ -221,6 +221,39 @@ def test_interface_fec_config(self):
...
+        self.assertTrue(response.status_code == status.HTTP_200_OK)
+
+        self.assert_with_timeout_retry(
+            lambda path, payload: self.get_req(path, payload),
+            self.assertEqual,
+            "device_interface_list",
+            {"mgt_ip": device_ip, "name": ether_name},
+            autoneg=set_autoneg,
+            status=status.HTTP_200_OK,
+        )
```
Comment (kamalkrbh) [inline network/test/test_interface.py:255]: @vivek-metricdust At the end set the origional auto neg value and assert that it is correctly set.

**Review by kamalkrbh — COMMENTED** [review]
orca_nw_lib version should be in sync with new version.

**Review by kamalkrbh — CHANGES_REQUESTED** [review]
Have look at comments in the files.

File: network/test/test_interface.py (hunk 1 above)
Comment (vivek-metricdust) [reply network/test/test_interface.py:262]: fixes [16769ae](https://github.com/STORDIS/orca_backend/pull/33/commits/16769aec019dbc41712e73fdd15a0451f27c1173)

**Review by kamalkrbh — APPROVED** [review]

**Pull request merged**